src/components/error/ErrorBoundary.tsx

```diff
@@ -0,0 +1,27 @@
+// External libraries
+import { Component, FunctionComponent, ReactNode } from "react";
+
+class ErrorBoundary extends Component<
```

Comment:
UMM, class component

Reply:
Umm, functional component is not supported here.  At least, it doesn’t look like it.
See [documentation](https://nextjs.org/docs/advanced-features/error-handling).

Let me know though if you found a functional component example.